nautobot_floor_plan/forms.py

```diff
@@ -73,6 +78,7 @@ class FloorPlanTileForm(NautobotModelForm):
     rack = DynamicModelChoiceField(
         queryset=Rack.objects.all(), required=False, query_params={"nautobot_floor_plan_floor_plan": "$floor_plan"}
     )
+    _char = forms.CharField()  # temp char field for replacement in __init__ if grid numbering is using letters.
```

Comment:
Would it be better to just change the fields to CharField and cast to integers if the choice is `NUMBERS`?

Reply:
Currently the form allows only integers. If the field type is not changed, then it comes blank.
Changing to CharField may simplify that. I would need to check

Reply:
Yes, that helped.